packages/react-instantsearch-hooks/README.md

```diff
@@ -468,6 +468,132 @@ function Hits(props) {
 }
 ```
 
+### `useMenu`
+
+> `(props: UseMenuProps) => MenuRenderState`
+
+Hook to use a [menu](https://www.algolia.com/doc/api-reference/widgets/menu/js/).
+
+**Types**
+
+<details>
+<summary><code>UseMenuProps</code></summary>
+
+```ts
+type UseMenuProps = {
+  /**
+   * The name of the attribute in the records.
+   */
+  attribute: string;
+  /**
+   * The max number of items to display when
+   * `showMoreLimit` is not set or if the widget is showing less value.
+   */
+  limit?: number;
+  /**
+   * Whether to display a button that expands the number of items.
+   */
+  showMore?: boolean;
+  /**
+   * The max number of items to display if the widget
+   * is showing more items.
+   */
+  showMoreLimit?: number;
+  /**
+   * How to sort refinements. Possible values: `count|isRefined|name:asc|name:desc`.
+   *
+   * You can also use a sort function that behaves like the standard Javascript [compareFunction](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort#Syntax).
+   *
+   * If a facetOrdering is set in the index settings, it is used when sortBy isn't passed
+   */
+  sortBy?: SortBy<MenuItem>;
+  /**
+   * Function to transform the items passed to the templates.
+   */
+  transformItems?: TransformItems<MenuItem>;
```

Comment:
I'm wondering how helpful theses types are, written as is. 
I think seeing `(items: MenuItem[]) => MenuItem[]` would be more explicit, for documentation.
WDYT?

Reply:
good point, in some others this is "made explicit"